portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -87,5 +105,42 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Redirect back to the HTML page.
     response.sendRedirect("/comments.html");
   }
+
+  /** Returns a URL that points to the uploaded file, or null if the user didn't upload a file. */
```

Comment:
Follow the same java docs approach with @return

(all functions that need it)

Reply:
Did you see my comment on the other PR? In the style guide I think they recommend 'Returns' instead of @return when writing simple docs with one sentence.